repo.sh

```diff
@@ -184,8 +193,9 @@ reset ()
         name="${BASH_REMATCH[1]}"
 
         if [ -d "$name" ]; then
+            DEFAULT_BRANCH=$(git symbolic-ref refs/remotes/origin/HEAD | sed 's@^refs/remotes/origin/@@')
```

Comment:
```suggestion
            DEFAULT_BRANCH=$(cd "$name"; git symbolic-ref refs/remotes/origin/HEAD | sed 's@^refs/remotes/origin/@@')
```
In testing this out, it looks like this line fails because it's not happening in the context of the target git repo.

Reply:
(unlike `_checkout_and_update_branch`, this function isn't called within the context of a directory switch)